applications/paraport_m1.md

```diff
@@ -0,0 +1,308 @@
+# ParaPort – Revolutionizing Cross-Chain Experiences in the Polkadot Ecosystem
+
+
+- **Team Name:** ParaPort
+- **Payment Details:**
+  - **DOT**: 13pa5ASHLLYLbEqN8EK2CzJbJ4BxXGcJ7xnVAwkMN7dTEkmY
+  - **Payment**: USDC
+- **[Level](https://github.com/w3f/Grants-Program#baby_chick-level-2):** 2
+
+## Project Overview :page_facing_up:
+
+![image](https://hackmd.io/_uploads/H1AraVBAkx.png)
+
+### Overview
+
+**ParaPort** is an embedded auto-teleport solution designed to simplify cross-chain transactions in the Polkadot ecosystem. Its primary function is to detect when users need tokens on different parachains and seamlessly handle the entire teleportation process alongside the user's intended action with minimal additional friction. Originally developed as an internal feature within [KodaDot](https://kodadot.xyz/) for minting NFTs on AssetHub, ParaPort has evolved into a comprehensive plug-and-play integration that any application in the Polkadot ecosystem can implement, bridging the gap between complex XCM mechanisms and intuitive user experiences without requiring extensive technical knowledge from developers or users.
+
+ParaPort is based on the academic research paper ["Protocol for Unifying Cross-Chain Liquidity on Polkadot" (Valaštín et al., 2024)](https://www.frontiersin.org/journals/blockchain/articles/10.3389/fbloc.2024.1413840/full). The building blocks implement *ParaSpell*, an intelligent XCM library, the most efficient way for moving tokens between chains. As a result, developers no longer need to manually build or maintain complicated cross-chain logic, and users no longer need to guess how many tokens to bridge or which route to use.
+
+**Key Features of ParaPort include:**
+
+- Automatic detection of cross-chain token requirements
+- Streamlined processing that handles bridging before executing user-intended transactions in a seamless flow
+- Simple, customizable user interface requiring only signatures from the user
+- Readily embeddable with minimal configuration
+
+By simplifying cross-chain operations, ParaPort aims to increase the adoption of Polkadot parachains, create a more cohesive user experience across the ecosystem, and enable new possibilities for cross-chain applications.
+![image](https://hackmd.io/_uploads/By0pnES0kx.png)
```

Comment:
Could you replace these links by pasting the images into the GH editor (it will create a githubusercontent.com/... link); these don't seem to render. Also, we prefer to have all files in one place.

Reply:
I am sorry, my bad. Updated now